webhook/resources/backuptarget/validator.go

```diff
@@ -0,0 +1,276 @@
+package backuptarget
```

Comment:
I think we should also do following check here
```
User can not delete or disable a default backup target directly.
```

Reply:
Added this logic at the validator.